diff --git a/node_modules/react-native-tts/android/.gradle/9.2.0/checksums/checksums.lock b/node_modules/react-native-tts/android/.gradle/9.2.0/checksums/checksums.lock
new file mode 100644
index 0000000..abdb764
Binary files /dev/null and b/node_modules/react-native-tts/android/.gradle/9.2.0/checksums/checksums.lock differ
diff --git a/node_modules/react-native-tts/android/.gradle/9.2.0/fileChanges/last-build.bin b/node_modules/react-native-tts/android/.gradle/9.2.0/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-tts/android/.gradle/9.2.0/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.bin b/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.bin
new file mode 100644
index 0000000..1856c6f
Binary files /dev/null and b/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.bin differ
diff --git a/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.lock b/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..b322913
Binary files /dev/null and b/node_modules/react-native-tts/android/.gradle/9.2.0/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-tts/android/.gradle/9.2.0/gc.properties b/node_modules/react-native-tts/android/.gradle/9.2.0/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..57e78df
Binary files /dev/null and b/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..0c89ed4
--- /dev/null
+++ b/node_modules/react-native-tts/android/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Wed Sep 23 04:53:17 CST 2026
+gradle.version=9.2.0
diff --git a/node_modules/react-native-tts/android/.gradle/vcs-1/gc.properties b/node_modules/react-native-tts/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-tts/android/build.gradle b/node_modules/react-native-tts/android/build.gradle
index dbbe046..22d5c53 100644
--- a/node_modules/react-native-tts/android/build.gradle
+++ b/node_modules/react-native-tts/android/build.gradle
@@ -4,7 +4,8 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
     }
 
     dependencies {
